.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules
 .DS_Store
 npm-debug.log
 dist
+tmp/less-lint-output.txt
```

Comment:
nit: missing newline at the end
also, this can be avoided if we output to `/tmp/...` instead (comment below)